docusaurus.config.js

```diff
@@ -76,8 +76,12 @@ const config = {
       title: "hackmud Wiki",
     },
     footer: {
-      copyright: `Copyright © ${new Date().getFullYear()} ComCODE`,
-      links: [],
+      logo: {
+        alt: "License: CC BY-NC-SA 4.0",
+        src: "https://img.shields.io/badge/License-CC_BY--NC--SA_4.0-lightgrey.svg",
+        href: "https://creativecommons.org/licenses/by-nc-sa/4.0/",
+      },
+      copyright: `Licensed for creative and educational use by ComCODE`,
```

Comment:
In terms of style, probably don't need a backtick string here now that there is no longer an expression inside.

Reply:
We also briefly mentioned in voice considering moving the shield image into the copyright string (which supports HTML) rather than using the logo field.

Reply:
updated.

Reply:
What did you think of swapping the backtick string to regular quotes?

Reply:
Oh never mind, it's got double-quotes on the inside. I guess personal preference between backtick and single-quote then.